File: TugasPraktikum05/Bahasa Python/PRAK501-2310817320013-AikoAnatashaWendiono.py
def MaxBilangan(a, b, c, d):
    bil = a

    if (b > bil):
        bil = b

    if (c > bil):
        bil = c

    if (d > bil):
        bil = d

    return bil

def bilangan():
    a, b, c, d = map(int, input().split())
    hasil = MaxBilangan(a, b, c, d)
    print(hasil)

bilangan()

File: TugasPraktikum05/Bahasa C/PRAK501-2310817320013-AikoAnatashaWendiono.c
#include <stdio.h>

int MaxBilangan(int a, int b, int c, int d){

    int bil = a;

    if (b > bil){
        bil = b;
    }

    if (c > bil){
        bil = c;
    }

    if (d > bil){
        bil = d;
    }

    return bil;
}


int main(){
int a, b, c, d; 
scanf("%d %d %d %d", &a, &b, &c, &d);
int hasil = MaxBilangan(a, b, c, d);
printf("%d", hasil); 
return 0;
}
